### ISSUE TITLE
Importing of completed NZBGet downloads stall when some unimportable files exist. 

### ISSUE THREAD

**Issue by desimaniac** [Sonarr/Sonarr#2568]
### System Information/Logs

**Sonarr Version:**  2.0.0.5203 develop

**Operating System:** Ubuntu Server LTS 16.04

**.net Framework (Windows) or mono (macOS/Linux) Version:** 5.10.1.20

**Link to Log Files (debug or trace):** 

Trace:
https://paste.ee/p/xtbUx
https://paste.ee/p/m4vyb

Debug:
https://paste.ee/p/6DJEX

### Additional Information

So I had a large amount of episodes finish downloading in NZBGet. All download with Sonarr via "search missing episodes" button. Once those downloads completed, importing stops and they remain under the `Success` tab in NZBGet ( a good 1000+). 

https://i.imgur.com/FEnE2KW.png

Sonarr will show that some of those have issues importing, either due to "no eligible file to import" or "file already exists" etc. 

https://i.imgur.com/ksy2kBM.png

Once a few of those are fixed via Sonarr's Activity - Queue page, the importing will resume for a good amount of imports before it stops again and requiring more items to be dealt with. 

### What problem are you looking to solve?

Basically, skip the imports it cant move, but still allow the rest of them to complete.

**Comment by markus101** [comment]
Sonarr limits the number of items it processes from NZBGet's history, otherwise not deleting the history would lead to slower and slower processing times.

The default limit is 30 items. If you want to increase the limit you can do so in the SQLite DB, by adding an entry to the `Config` table. `INSERT INTO Config (Key, Value) VALUES ('downloadclienthistorylimit', 60)` will increase it to 60.

**Comment by desimaniac** [comment]
Understood. Thanks!
